src/config/cfg_rules.ini

```diff
@@ -332,6 +336,58 @@ option = scope
 option = users
 option = groups
 
+# Prompting during authentication
+[rule/allowed_prompting_password_options]
+validator = ini_allowed_options
+section_re = ^prompting/password$
+
+option = timeout
+option = debug
+option = debug_level
+option = debug_timestamps
+option = debug_microseconds
+option = debug_to_files
+option = command
+option = reconnection_retries
+option = fd_limit
+option = client_idle_timeout
+option = description
+option = password_prompt
```

Comment:
Hi,

currently only `password_prompt` is allowed in the `[prompting/password]` section, see sssd.conf man page. Since this section does not relate to a dedicated process the common options like e.g. `timeout` or `debug_level` are not available.